app/common/footer/footer.template.html

```diff
@@ -36,9 +53,20 @@
         <img src="/assets/img/gapminder-logo.png" alt="Gapminder">
       </a>
     </div>
+
+    <div class="valor">
+      <div class="developed">Developed by</div>
+
+      <a style="color: #e7464d" href="http://valor-software.com" target="_blank"
+         angulartics2On="click"
+         angularticsEvent="Moved to Valor Software">
+        Valor-Software
+      </a>
+    </div>
+
   </div>
 
   <div class="social-share-buttons-container">
```

Comment:
@daniloff200 rename class `social-share-buttons-container` it's not social share buttons it is follow buttons